qt/python/mantidqt/project/encoderfactory.py

```diff
@@ -30,7 +32,12 @@ def find_encoder(cls, obj):
         elif len(obj_encoders) == 1:
             return obj_encoders[0]()
         else:
-            return None
```

Comment:
Is there a reason we `return None` in this case but raise an exception in the `DecoderFactory`?